samples/algorithms/BellState.qs

```diff
@@ -7,14 +7,12 @@
 ///
 /// This Q# program implements a Bell state.
 namespace Sample {
-    open Microsoft.Quantum.Intrinsic;
     open Microsoft.Quantum.Diagnostics;
 
     @EntryPoint()
     operation BellState() : (Result, Result) {
         // Allocate the two qubits that will be used to create a Bell state.
-        use q1 = Qubit();
-        use q2 = Qubit();
+        use (q1, q2) = (Qubit(), Qubit());
```

Comment:
BellState is one of the first samples we have. Do we want to use tuples right from the start? I think it may be easier to understand the very first sample without tuples. On the other hand, they don't seem to be too complex...

Reply:
I think they're not too complex and seem easy to understand. They work the same way that tuples in other languages so it doesn't seem like a big deal IMO.

Reply:
I don't mind using tuples here especially if you add Superposition sample as the first sample.